### DISCUSSION TITLE
FLTK 1.4 released

### DISCUSSION THREAD

**Discussion by robark** [Homebrew/discussions#5743] [Tap maintenance and Homebrew development]
### Output of `brew config`

```shell
brew config
```

### Output of `brew doctor`

```shell
brew doctor
```

### Description of issue

new version of FLTK gui library released
macOS is supported up to 15.0 "Sequoia". 
https://www.fltk.org/articles.php?L1955

**Comment by SMillerDev** [comment]
https://docs.brew.sh/Updating-Software-in-Homebrew

**Comment by daeho-ro** [comment]
We have some blocking issue here,
- https://github.com/Homebrew/homebrew-core/pull/198029
